### PR TITLE
Placeholder script to import hubspot data

### PR DESCRIPTION
**Pull request by marcospri**
For https://github.com/hypothesis/report/issues/39

### CODE REVIEW

File: .github/workflows/imports.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+on:
+  workflow_dispatch:
+    inputs:
+      Task:
+        type: choice
+        description: "The task to perform"
+        required: true
+        options:
+          - 'hubspot'
```
Comment (marcospri) [inline .github/workflows/imports.yml:13]: I reckon we don't want a worflow for hubspot and another for let's say zendesk, etc.

File: .github/workflows/imports.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  hubspot:
+    needs: check_for_danger
+    name: "Imports hubspot data into the report's DB"
+    uses: hypothesis/workflows/.github/workflows/eb-task.yml@main
+    with:
+      App: ${{ github.event.repository.name }}
+      Env: ${{ inputs.Environment }}
+      Timeout: 3600
+      Region: 'us-west-1'
```
Comment (marcospri) [inline .github/workflows/imports.yml:47]: This is US only

File: .github/workflows/imports.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+          - 'qa'
+          - 'prod'
+      Destructive:
+        description: Check to confirm you are happy to proceed with a destructive operation
+        type: boolean
+        required: false
+        default: false
+
+jobs:
+  check_for_danger:
```
Comment (marcospri) [inline .github/workflows/imports.yml:29]: I reckon having the chekbox it good regardless? It's doing something in production.

We can always remove it.

**Review by jon-betts — APPROVED** [review]
Looks good, but it probably requires checking like all CI stuff does.

**Pull request merged**